doit.sh

```diff
@@ -198,6 +198,9 @@ time openstack undercloud install --experimental \\
 EOF_CAT
 chmod 755 $HOME/run.sh
 
+if [ ! -f $HOME/undercloud.conf ]; then
+    cp $(dirname "${BASH_SOURCE[0]}")/undercloud.conf $HOME/undercloud.conf
```

Comment:
doit.sh is currently self-contained though. It isn't a guarantee that users would have undercloud.conf on the same machine. So perhaps we could git-fetch or cat out just a minimal version? I honestly don't think the full version of undercloud.conf belongs in this dev repo anyway.

Reply:
Well, it was already in the repository and I couldn't figure out a better place to fetch it from :/